crates/cdk/src/cdk_database/mod.rs

```diff
@@ -102,9 +102,9 @@ pub trait MintDatabase {
     async fn get_melt_quotes(&self) -> Result<Vec<MeltQuote>, Self::Err>;
     async fn remove_melt_quote(&self, quote_id: &str) -> Result<(), Self::Err>;
 
-    async fn add_keyset(&self, keyset: MintKeySet) -> Result<(), Self::Err>;
-    async fn get_keyset(&self, id: &Id) -> Result<Option<MintKeySet>, Self::Err>;
-    async fn get_keysets(&self) -> Result<Vec<MintKeySet>, Self::Err>;
+    async fn add_keyset(&self, keyset: MintKeySetInfo) -> Result<(), Self::Err>;
+    async fn get_keyset(&self, id: &Id) -> Result<Option<MintKeySetInfo>, Self::Err>;
+    async fn get_keysets(&self) -> Result<Vec<MintKeySetInfo>, Self::Err>;
```

Comment:
Should these be renamed to `add_keyset_info` etc or think its okay to leave them?

Reply:
I can add `_info` to make it more explicit.